docs/getting_started/import_automation.md

```diff
@@ -21,384 +21,240 @@ Notizen:
-    ShortDisplayText = "Checking PiWeb",
-    DetailedDisplayText = $"Checking PiWeb for {targetPath}"
+    _ActivityService.PostActivityEvent(EventSeverity.Error, "The import target is not supported");
+    return;
   }
-);
-```
 
-![Auto Importer events](../../assets/images/getting_started/import_automation/events.png "Auto Importer events")
-
-Now we request the PiWeb Cloud instance using our part structure. This returns the existing part if it is already known, otherwise null.
+  try
+  {
+    var uri = new Uri(_ImportTarget.ServiceAddress);
+    var authData = _ImportTarget.AuthData;
+    using var restClient = CreateDataServiceClient(uri, authData);
+    var random = new Random();
 
-```c#
-// Request PiWeb API and check for part
-var knownParts = await restClient
-                        .GetParts(targetPath, depth: 0, cancellationToken: cancellationToken)
-                        .ConfigureAwait(false);
-var targetPart = knownParts.FirstOrDefault();
-```
+    while (!cancellationToken.IsCancellationRequested)
+    {
+      var value = Math.Round(random.NextDouble(), 2);
 
-If our part is available, we display this accordingly in the Auto Importer. If it does not exist, the PiWeb Server is informed that it should be created.
+      var targetPart = await GetOrCreateTargetPart(restClient);
+      var characteristic = await GetOrCreateCharacteristic(restClient, targetPart);
+      await UploadMeasurement(restClient, targetPart, characteristic, value);
 
-```c#
-if (targetPart != null)
-{
-  // Part is known in database
+      _ActivityService.PostActivityEvent(EventSeverity.Info, $"Uploaded new measurement value: {value}");
 
-  _statusService.SetActivity(
-    new ActivityProperties()
-    {
-      ActivityType = ActivityType.Normal,
-      ShortDisplayText = "Part exists",
-      DetailedDisplayText = $"{targetPath} exists in database"
+      await Task.Delay(TimeSpan.FromSeconds(5), cancellationToken);
     }
-  );
-}
-else
-{
-  // Part is unknown in database
-
-  _statusService.SetActivity(
-    new ActivityProperties()
-    {
-      ActivityType = ActivityType.Suspension,
-      ShortDisplayText = "Part does NOT exists",
-      DetailedDisplayText = $"{targetPath} not found in database, creating it"
-    }
-  );
-
-  // Create that part
-  var part = new InspectionPlanPartDto
+  }
+  catch (OperationCanceledException)
   {
-    Uuid = Guid.NewGuid(),
-    Path = targetPath
-  };
-
-  await restClient.CreateParts([part], cancellationToken: cancellationToken).ConfigureAwait(false);
+    // Do nothing
+  }
 }
 ```
 
-As a final action, we delay the following loop pass by 5 seconds to keep the load on the instance low.
+First we check whether the backend connection configured for the import plan is actually a web service. Since we use the *Piweb API* to connect to the backend, we do not support any other connection types. Next, we create the rest client using the `CreateDataServiceClient` method from step 5.

```

Comment:
There is a mix of "API" and "Api" notation in the document. We should standardize this globally.

Reply:
Let's use "PiWeb-Api" since that is how we call it on github.